prime-dotnet-webapi/Services/CommunitySiteService.cs

```diff
@@ -45,6 +46,31 @@ public async Task<IEnumerable<CommunitySite>> GetSitesAsync(int? organizationId
             return await query.ToListAsync();
         }
 
+        public async Task<PaginatedList<CommunitySiteAdminListViewModel>> GetSitesAsync(OrganizationSearchOptions searchOptions)
+        {
+            searchOptions ??= new OrganizationSearchOptions();
+
+            var query = _context.CommunitySites
+                .AsNoTracking()
+                .If(searchOptions.CareSettingCode.HasValue, q => q
+                    .Where(s => s.CareSettingCode == searchOptions.CareSettingCode)
+                )
+                .If(!string.IsNullOrWhiteSpace(searchOptions.TextSearch), q => q
+                    .Search(
+                        s => s.DoingBusinessAs,
+                        s => s.PEC,
+                        s => s.Organization.Name,
+                        s => s.Organization.DisplayId.ToString(),
+                        s => s.Organization.SigningAuthority.FirstName + " " + s.Organization.SigningAuthority.LastName)
+                    .Containing(searchOptions.TextSearch)
+                )
+                .ProjectTo<CommunitySiteAdminListViewModel>(_mapper.ConfigurationProvider)
+                .OrderBy(s => s.DisplayId)
```

Comment:
`OrderBy` helped but still Organization should come before its Sites

![image](https://user-images.githubusercontent.com/7586665/160896127-f9f06ef6-2231-4fe0-b086-f1b9da982a85.png)

Reply:
OK, never mind about comment re: "Organization should come before its Sites" but the Sites within an Org should be sorted more intuitively, by Submission Date or whatever logic was implemented before?